INN/umbrella-ipbs
# Navigation and header

## Changes

This pull request makes the following changes:

- Adds styling and functionality for the site navigation and header, as well as the mobile nav

Desktop:
![Screen Shot 2020-01-24 at 4 05 18 PM](https://user-images.githubusercontent.com/18353636/73104017-69a22400-3ec3-11ea-8c00-46df6f276c15.png)

Mobile closed:
![Screen Shot 2020-01-24 at 4 05 31 PM](https://user-images.githubusercontent.com/18353636/73104016-69098d80-3ec3-11ea-83cb-5760a543bec6.png)

Mobile open:
![Screen Shot 2020-01-24 at 4 05 37 PM](https://user-images.githubusercontent.com/18353636/73104013-69098d80-3ec3-11ea-93ea-13690b8fcbf0.png)

## Why



For #43 and #44 

## Testing/Questions

Features that this PR affects:

- Nav and header


Questions that need to be answered before merging:

- [x] Can anything else be done in this PR with the designs we have so far?

Steps to test this PR:

1. Add some header images via theme options
![ipbs](https://user-images.githubusercontent.com/18353636/72914987-8e549b00-3d0d-11ea-8a31-8fdb0d861cdc.png)
2. Uncheck `Use only text in the place of a banner image (uses site title and description).`
3. Disable sticky navigation
4. Create a menu and add it to the `Main Navigation` nav location
5. Create a menu and add it to the `Main Navigation Bottom` nav location
6. View on the frontend and verify it looks ok
